src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "024")
```

Comment:
This needs to be the next highest number in the `ChangeLog` sequence.

Reply:
updated, set to 48.

Reply:
Sorry, I think the next highest is 49.

Reply:
@maxandersen I had to fix a defunct migration on master, so your branch needs a rebase, sorry. Also, with all the incoming traffic, can you bump your changelog to 53, please. I feel like an air traffic controller :stuck_out_tongue: